frontend/src/components/orgs-list.ts

```diff
@@ -593,16 +597,82 @@ export class OrgsList extends BtrixElement {
       };
     }
 
+    let subscription = {
+      icon: none,
+      description: msg("No Subscription"),
+    };
+
+    if (org.subscription) {
+      switch (org.subscription.status) {
+        case SubscriptionStatus.Active:
+          subscription = {
+            icon: html`<sl-icon
+              class="text-base text-success"
+              name="nut-fill"
+              label=${msg("Active Subscription")}
+            ></sl-icon>`,
+            description: msg("Active Subscription"),
+          };
+          break;
+        case SubscriptionStatus.Trialing:
+          subscription = {
+            icon: html`<sl-icon
+              class="text-base text-neutral-400"
+              name="nut"
```

Comment:
Basket because the customer is still shopping around, credit card for active billing? 🙃 

```suggestion
              name="basket-fill"
```

Reply:
I used `nut` because that's what we use in the Org Settings page next to subscription info but I'm open to whatever you think is best!

Reply:
The nut in org settings is supposed to be for the basic subscription plan ("nuts and bolts" only) but we should probably update that since we have more tiers now!

Reply:
Ah, okay! Yeah in this case we're not showing which plan they're on in the orgs list, just whether they're susbscribed, but we could potentially also show a different icon per plan type?